ledger/acctonline.go

```diff
@@ -491,6 +546,26 @@ func (ao *onlineAccounts) roundOffset(rnd basics.Round) (offset uint64, err erro
 	return off, nil
 }
 
+// roundParamsOffset calculates the offset of the given round compared to the onlineRoundParams cache. Requires that the lock would be taken.
+func (ao *onlineAccounts) roundParamsOffset(rnd basics.Round) (offset uint64, err error) {
+	dbRound := ao.latest() + 1 - basics.Round(len(ao.onlineRoundParamsData))
```

Comment:
please use another name, `dbRound` usually means "tracker db round" and not the deepest history entry round